classes/plugins/PluginHelper.inc.php

```diff
@@ -45,7 +45,13 @@ function extractPlugin($filePath, $originalFileName, &$errorMsg) {
 		// equal plugin directory name and plugin files must be in a
 		// directory named after the plug-in (potentially with version)
 		$matches = array();
-		String::regexp_match_get('/^[a-zA-Z0-9]+/', basename($originalFileName, '.tar.gz'), $matches);
+		$path_parts = pathinfo($originalFileName);
```

Comment:
I think this code should be a private method so it does not have to be repeated again on https://github.com/pkp/pkp-lib/pull/751/files#diff-192390af7614d51a83a27e64641372b6R83